atspi-common/src/events/object.rs

```diff
@@ -109,17 +124,17 @@ impl TryFrom<EventBody<'_>> for Property {
 		let property = body.kind();
 
 		match property {
-			"accessible-name" => Ok(Self::Name(
+			a if a == ACCESSIBLE_NAME_PROPERTY_NAME => Ok(Self::Name(
```

Comment:
```suggestion
			ACCESSIBLE_NAME_PROPERTY_NAME => Ok(Self::Name(
```

Please also update all match arms below.